src/createContext.ts

```diff
@@ -205,6 +206,11 @@ function importPrelude(context: Context) {
   if (context.chapter >= 3) {
     prelude += streamPrelude
   }
+
```

Comment:
See discussion in group "SA js-slang-related", April 2 2020:

Martin Henz, [2 Apr 2020 at 3:17:09 PM]:
Folks, we need to find a solution for this. I just saw this in the non-det PR:

if (context.chapter === 4.3) {
    prelude += nonDetPrelude
  }
I don't think we should use floating point numbers for this clearly discrete issue.

From the user point of view, I'm suggesting that we keep integer chapter numbers: 1-4, but we add modifiers, perhaps we can call them "variants".

So we would have in the drop-down:

Source §1
Source §2
Source §2 Lazy
Source §3 Non-det
Source §3 Concurrent
Source §4
and eventually

Source §x Arduino
Source §x Wasm

Under the hood, we currently have:
Source §1: native (default)
Source §1: small-step evaluator (when stepper tool is chosen)
Source §2: native (default)
Source §2: small-step evaluator (when stepper tool is chosen)
Source §3: native (default)
Source §3: interpreter (when debugger/env viz are chosen)
Source §4: native (default)
Source §4: interpreter (when debugger/env viz are chosen)

We are proposing to add:
Source §2 Lazy: native
Source §3 Non-det: interpeter
Source §3 Concurrent`: virtual machine